README.md

```diff
@@ -4,3 +4,26 @@ MetalKube Ironic Container
 This repo contains the files needed to build the Ironic images used by metalkube.
 
 When updated, builds are automatically triggered on https://quay.io/repository/metalkube/metalkube-ironic/
+
+This repo supports the creation of multiple containers needed when provisioning baremetal nodes with Ironic. Eventually there will be separate images for each container, but currently separate containers can share this same image with specific entry points.
+
+The following entry points are provided:
+- runironic - Starts the ironic-conductor and ironic-api processes to manage the provisioning of baremetal nodes.  Details on Ironic can be found at https://github.com/openstack/ironic.  This is the default entry point used by the Dockerfile.
```

Comment:
nit: github is just a mirror, I guess the canonical URL is https://docs.openstack.org/ironic/latest/

Reply:
Also I'd prefer the lines wrapped

Reply:
++ We have very nice documents in a rendered form for those curious as opposed to being dropped in a code repo.

Reply:
Fixed in https://github.com/metalkube/metalkube-ironic/pull/33